TCMSApp/src/server/app.js

```diff
@@ -3,25 +3,56 @@
 
 var express = require('express');
 var app = express();
+var methodOverride = require('method-override');
+var mongoose = require('mongoose');
+var restify = require('express-restify-mongoose');
 var bodyParser = require('body-parser');
 var favicon = require('serve-favicon');
 var logger = require('morgan');
 var port = process.env.PORT || 8001;
 var four0four = require('./utils/404')();
 
+var docs = require("express-mongoose-docs");
+
 var environment = process.env.NODE_ENV;
 
+
+var router = express.Router();
+
 app.use(favicon(__dirname + '/favicon.ico'));
 app.use(bodyParser.urlencoded({extended: true}));
 app.use(bodyParser.json());
+app.use(methodOverride())
```

Comment:
As I understood it is used to simulate methods which are not supported by API and we don't need it for our API.
My bad.
I was experimenting with docs of express-restify-mongoose and added this piece automatically.
Removed.